buzzmobile/process/bearing/bearing.py

```diff
@@ -25,10 +25,10 @@ def median(self):
             return (sortedArray[index] + sortedArray[index + 1]) / 2.0
 
 
-#Global Variables
-bearings = {}
-bearings['last_fix'] = None
-bearings['med_filter'] = MedianFilter(rospy.get_param('median_filter_size'))
+# Global variables
+globalVars = {}
+globalVars['last_fix'] = None
+globalVars['med_filter'] = MedianFilter(rospy.get_param('median_filter_size'))
```

Comment:
OK cool. I haven't done Python before so please tell me of any stylistic or whatever errors. I understand what the dict was doing when Dip2Dip originally wrote it when he had it kinda badly named as bearings, but when you change the name to `globals ` it is highlighted by the IDE? Is `globals` a reserved word or a special dict or something?